demo.md

```diff
@@ -27,7 +27,7 @@ Before proceeding to the section of your choice, please consider the following t
 * You can use any 3rd party libraries you wish (`Alamofire`, `ReactiveCocoa`, `PromiseKit`, `Realm`, etc) but be prepared to justify why you did so. Feel free to use package managers to handle them.
 * **Be consistent in your code**. We advise using something like [raywenderlich's swift style guide](https://github.com/raywenderlich/swift-style-guide) while doing the demo. It's absolutly fine to use any other style, as long as you are consistent.
 * Clean the file project structure and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
-* If you have to use VIPER please give it your own spin. Even though it's a good architecture often times it carries a lot of boilerplate which makes all tests that use it look the same.
+* Be opinionated regarding any architecture you use, take your time to make it a little yours and a reflection of your logical thought.
```

Comment:
"Be opinionated regarding any architecture you use, take your time to make it a little yours and a reflection of your logical thought."  -> "Be opinionated regarding any architecture you use and take your time to make it a reflection of your thought process."